src/components/form-elements/draft-js-mention-selector/MentionSelector.scss

```diff
@@ -16,9 +16,18 @@
         position: absolute;
         z-index: 90;
         width: 100%;
-        padding: 8px;
+        margin-top: $bdl-grid-unit;
+        padding: $bdl-grid-unit * 2;
         background: $white;
         border: 1px solid $bdl-gray-30;
+        border-radius: $bdl-border-radius-med-size;
         box-shadow: 0 2px 6px 0 fade-out($black, .9);
     }
+
+    // Normally .bdl-SelectorDropdown > .text-input-container has a bottom margin
+    // but the DraftJSEditor doesn't have one.
+    // Add spacing above dropdown wrappers here instead:
+    .overlay {
```

Comment:
Is this a bug fix? If so, should it be a separate PR?

Reply:
It aligns the spacing of this element, which customizes the SelectorDropdown with an alternative text input box, with the styling of the other selectors.

<img width="336" alt="Screen Shot 2021-11-08 at 5 33 33 PM" src="https://user-images.githubusercontent.com/1571667/141006292-0b52d3e3-5b50-4444-821e-d91016770e22.png">


<img width="335" alt="Screen Shot 2021-11-09 at 1 20 42 PM" src="https://user-images.githubusercontent.com/1571667/141006869-3d6fed1b-13a9-4cb0-8ee3-8f1555942a46.png">